x/emissions/keeper/inference_synthesis/network_inferences.go

```diff
@@ -346,7 +392,7 @@ func CalcNetworkInferences(
 	}
 
 	// Calculate the one-out inference I^-_li
-	oneOutInferences, oneOutImpliedInferences, err := CalcOneOutInferences(ctx, k, topicId, inferenceByWorker, forecastImpliedInferenceByWorker, forecasts, maxCombinedRegret, networkCombinedLoss, epsilon, pInferenceSynthesis)
+	oneOutInferences, oneOutImpliedInferences, err := CalcOneOutInferences(ctx, k, topicId, inferenceByWorker, forecasts, maxCombinedRegret, networkCombinedLoss, epsilon, pInferenceSynthesis)
```

Comment:
I don't agree with removing the forecastImpliedInference argument for this reason -- computation.

It's already computed and in memory => should pass it in. Yes it could be recreated, but it shouldn't be if it already exists.

Maybe this admits slightly less sexy code, but this is an expensive computation as far as on-chain computations go, so I think we need to be conservative.